README.md

```diff
@@ -336,11 +334,15 @@ in [RFC 8174](https://rfc-editor.org/info/rfc8174).
 [BCP 78](https://www.rfc-editor.org/info/bcp78) and [BCP
 79](https://www.rfc-editor.org/info/bcp79) specify the IETF's Intellectual
 Property policies. Among them is a requirement to include certain standardized
-text in each Internet-Draft. This text is managed by the IAB, see the [current
-acceptable boilerplate
-texts](https://www.iab.org/documents/headers-boilerplate/) and the [fully
-expanded forms of the boilerplate
-texts](https://www.rfc-editor.org/materials/status-memos.txt).
+"boilerplate" text in each Internet-Draft. This text is managed by the IETF
+Trust, see [TLP Section
+6](https://trustee.ietf.org/documents/trust-legal-provisions/).
+
+If you think that you, your company, or anyone else owns a patent or other
+Intellectual Property Rights (IPR) on the work described in the I-D, you should
+carefully read [BCP 79](https://www.rfc-editor.org/info/bcp79). The part of the
+required standardized text in an I-D obligates you to send an
+IPR disclosure statement under many circumstances.
```

Comment:
```suggestion
If you think that you, your company, or anyone else owns a patent or other
Intellectual Property Rights (IPR) on the work described in the I-D, you should
carefully read [BCP 79](https://www.rfc-editor.org/info/bcp79). The first part of the
required standardized text in an I-D obligates you to send an
IPR disclosure statement under many circumstances.
```

Reply:
@kaduk good catch. This text existed in the document before (scroll down to where I moved it from), but I edited it some and that caused the issue.